doc/contributing/technical-priorities.md

```diff
@@ -121,3 +108,34 @@ components/approaches for doing this, they need to be better
 documented and evangelized so that this is not seen as a barrier
 for using Node.js in these situations. This is important to support
 the expansion of where/when Node.js is used in building solutions.
+
+## Serverless
+
+Serverless is a cloud computing model where the cloud provider manages the
+underlyinginfrastructure and automatically allocates resources as
+needed. Developers only need to focus on writing code for specific
+functions, which are executed as individual units of work in response to
+events. Node.js is one of the main technology used by developers in
+this field therefore it is crucial for us to provide a great solution.
+
+## Small footprint
+
+Small software footprints refer to software that has a minimal impact on
+system resources such as memory and processing power. This can be achieved
+through various methods such as optimizing code, reducing the number of
+dependencies, or using lightweight frameworks. Smaller footprints can lead
+to faster startup times, reduced memory usage, and improved overall system
+performance. This is fundamental for Node.js to be a lightweight proposition
+inside the ecosystem as it is used across a wild variety of projects, from
+web application to IoT and serverless.
+
+## Developers-first DX
```

Comment:
I think this should mention TS Support, or TS support should be its own bucket.

Reply:
added a quick sentence. I think if ts will be one main point on the next 10 survey this year we can move that to its own dedicated part